app/util/CORSable.scala

```diff
@@ -0,0 +1,20 @@
+package util
+
+import play.api.mvc.{Action, BodyParser, Request, Result}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+case class CORSable[A](origins: String*)(action: Action[A]) extends Action[A] {
```

Comment:
Is there any reason this couldn't extend AuthAction? Doesn't need to be done in this PR but is is looking like we will need CORS on the root endpoint in the future for panda reauth to work correctly